src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -481,15 +481,18 @@ public function newInstance($attributes = array(), $exists = false)
 	 * Create a new model instance that is existing.
 	 *
 	 * @param  array  $attributes
+	 * @param  string  $connection
```

Comment:
It's not like that throughout the codebase, but... heck if that makes you happy :)